specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-05-01-preview/Migrations.json

```diff
@@ -0,0 +1,826 @@
+          "CheckMigrationNameAvailability": {
+            "$ref": "./examples/CheckMigrationNameAvailability.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/TargetDBServerSubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/TargetDBServerResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/TargetDBServerNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MigrationNameAvailabilityResource"
+            },
+            "description": "The required parameters for checking if a migration name is available."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MigrationNameAvailabilityResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "MigrationResource": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/MigrationResourceProperties",
+          "description": "Migration resource properties.",
+          "x-ms-client-flatten": true
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/systemData",
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Represents a migration resource."
+    },
+    "MigrationResourceProperties": {
+      "type": "object",
+      "properties": {
+        "migrationId": {
+          "type": "string",
+          "readOnly": true
+        },
+        "migrationName": {
+          "type": "string",
+          "readOnly": true
+        },
+        "migrationDetailsLevel": {
+          "$ref": "#/definitions/MigrationDetailsLevel",
+          "readOnly": true
+        },
+        "currentStatus": {
+          "$ref": "#/definitions/MigrationStatus",
+          "readOnly": true
+        },
+        "sourceDBServerMetadata": {
+          "$ref": "#/definitions/DBServerMetadata",
+          "readOnly": true
+        },
+        "targetDBServerMetadata": {
+          "$ref": "#/definitions/DBServerMetadata",
+          "readOnly": true
+        },
+        "sourceDBServerResourceId": {
```

Comment:
>sourceDBServerResourceId [](http://example.com/codeflow?start=9&length=24)

please ensure you have a linked access check on this property in your RP manifest

Reply:
Added @pilor 

<img width="383" alt="image" src="https://user-images.githubusercontent.com/69746551/222053184-02589fa1-5a42-457d-958b-b1448052594c.png">